### PR TITLE
web-development COOKIE/SESSION Homework No.1~No.7

### PR DESCRIPTION
**Pull request by kojiyuu-123**
This pull request contains homework, in "web-development_COOKIE/SESSION".

To show my page correctly , please implement "index.php" at the top directory
( web_development_COOKIE_SESSION ).
The function I implemented in this program are below.

  1. If the user doesn't login , print the message in main-contents area.
  2. If the user logged in , the user name is shown at header bar.
  3. After 30 minutes, the user must login again.
  4. If the user checked the checkbox("Remember me") in login form, the column, username in login form, is filled.
  5. logout function.
  6. Refactor to the user can view like home page when transit other pages(users, courses, trainers).
  7. During 3days we don't have to login again if we don't logout.

This page has login form. Some users have been already resistered (details below).
  [username]: A [password]: abc
  [username]: B [password]: def
  [username]: C [password]: pqr

Please check it.
Thank you.

### CODE REVIEW

File: web_development_COOKIE_SESSION/src/courses/courses.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+var course_array=[
+	//{id:"",course name:"",Author:"",duration:"",created date:""}
+	{id:"123",course_name:"web",Author:"H",duration:"4days",created_date:"1998"},
+	{id:"456",course_name:"sql",Author:"L",duration:"1week",created_date:"1999"},
+	{id:"789",course_name:"framework",Author:"V",duration:"1month",created_date:"2000"}
+];
+
+function show_courses(){
```
Comment (ohralathe) [inline web_development_COOKIE_SESSION/src/courses/courses.js]: @kojiyuu-123 for `JS`, use camel case plz

Next time, you should check the convention for each language before start.

File: web_development_COOKIE_SESSION/src/helpers/route.php [hunk 2]
```diff
@@ -0,0 +1,218 @@
+<?php
+/*
+class Route {
```
Comment (ohralathe) [inline web_development_COOKIE_SESSION/src/helpers/route.php]: If not use, should remove it.

File: web_development_COOKIE_SESSION/src/home/index.php [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    	
+    	<div id="popup">
+	    	<form id="loginform" method="POST" action="/login" >
+	    		<p>login</p>
+	    		<p>username</p>
+	    		
+	    		<input type='text' id='user_name' name='username' 
+	    		<?php
+	    		if(!isset($_COOKIE['username'])){
+	    			echo ("placeholder='username'");
```
Comment (ohralathe) [inline web_development_COOKIE_SESSION/src/home/index.php]: it is not `placeholder`. It's the value

**Review by ohralathe — CHANGES_REQUESTED** [review]
@kojiyuu-123 I checked and have some comments.

I don't know what homework you have done. so I just check some of them.

Next time, plz clarify out what you have done plz

File: web_development_COOKIE_SESSION/src/courses/courses.js (hunk 1 above)
Comment (kojiyuu-123) [reply web_development_COOKIE_SESSION/src/courses/courses.js]: Sorry, I'll change the code to camel case in javascript code.

File: web_development_COOKIE_SESSION/src/helpers/route.php (hunk 2 above)
Comment (kojiyuu-123) [reply web_development_COOKIE_SESSION/src/helpers/route.php]: OK.

**Comment by ohralathe** [comment]
@kojiyuu-123 good job.
thank you for update.

close this PR plz

**Comment by kojiyuu-123** [comment]
Thank you for your checking and comments

**Pull request closed without merging**